tutorials/plot_point_spread.py

```diff
@@ -0,0 +1,147 @@
+"""
+.. _point_spread:
+
+Corrupt known signal with point spread
+======================================
+
+The aim of this tutorial is to demonstrate how to put a known signal at a
+desired location(s) in a SourceEstimate and then corrupt the signal with
+point-spread by applying a forward and inverse solution.
+"""
+
+import os.path as op
+
+import numpy as np
+
+import mne
+from mne.datasets import sample
+
+from mne.minimum_norm import read_inverse_operator, apply_inverse
+from mne.simulation import simulate_stc, simulate_evoked
+
+###############################################################################
+# Parameters
+
+seed = 8675309
```

Comment:
Well, there's not actually any randomness in the current example, it's only if someone were to modify how the signal was generated or set `evoked_snr` to a positive finite value that the seed would matter.